gtk4/Gir.toml

```diff
@@ -757,6 +762,11 @@ status = "generate"
     name = "set_sort_func"
     manual = true
 
+[[object]]
+name = "Gtk.Expression"
+status = "manual"
+final_type = true
```

Comment:
This is "provisory". I wonder if we should have a configuration in gir side, where you can define the trait to use instead of `glib::IsA<T>`. So that I can use here `IsExpression` . I think it would be nice & reduce a bunch of manual code. Although, no idea how feasible all of this :-) Any ideas?

Reply:
That would require the information in the .gir that there's a type hierarchy. Does that exist?

Reply:
Indeed. I will keep the `final_type = true` here for now to not re-write the *Builders manually as otherwise they will use `IsA<Expression>`, using a plain `&Expression` should be fine as the "sub-classes" implement Deref.